### ISSUE TITLE
No playback with .webm and .mp4 videos available

### ISSUE THREAD

**Issue by aozalevsky** [pedrobmarin/bbb-playback#40]
**Describe the bug**
I tried current master and it works like a charm. But only with recordings from the period when we didn't have .mp4 recordings [enabled](https://docs.bigbluebutton.org/2.2/customize.html#enable-playback-of-recordings-on-ios)

**To Reproduce**
webm only recording: https://live.fbb.msu.ru/playback/presentation/2.3/69e403e3cb8511e7348803b6a0d2e7bbf8596b42-1584436259674

mp4/webm recording:
works in the current-stable bbb-playback
https://live.fbb.msu.ru/playback/presentation/2.0/playback.html?meetingId=22b5c023928b2a0e4bc9d254ad591a2fbd055caf-1588158996864

but not in beta:
https://live.fbb.msu.ru/playback/presentation/2.3/22b5c023928b2a0e4bc9d254ad591a2fbd055caf-1588158996864

I also don't see any error messages in browser console so not sure where to look to resolve the issue.

**Comment by aozalevsky** [comment]
can confirm. I commented out 

**src/config.json**
removed "mp4"
```
  "medias": [
    "webm"
  ],
```

**components/screenshare/index.js**
```
    const sources = [
//      {
//        src: `${url}/deskshare/deskshare.mp4`,
//        type: `video/mp4`,
//      }, 
        {
        src: `${url}/deskshare/deskshare.webm`,
        type: `video/webm`,
      },
```
**components/video/index.js**
```
    const sources = [
//      {
//        src: `${url}/video/webcams.mp4`,
//        type: 'video/mp4',
//      }, 
      {
        src: `${url}/video/webcams.webm`,
        type: 'video/webm',
      },

    ].filter(src => {
```

Now everything works as it should.

**Comment by pedrobmarin** [comment]
Hi, thanks for the pointing that out. I thought I had that case covered but it's possible I missed something there. Are you running it on something different then Firefox, Chrome or Safari?

**Comment by pedrobmarin** [comment]
And before I forget, BigBlueButton documentation lacks to inform that if you enable `mp4` at processing you really don't need to keep `webm` enabled too. This could save you some disk and cpu time at your recordings processors.
Although if your clients run Yandex Browser you will have to check if it supports h264/aac codecs. I believe it does.

**Comment by aozalevsky** [comment]
I use Chromium under Ubuntu.  Here is the full User-Agent string:

Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/81.0.4044.122 Chrome/81.0.4044.122 Safari/537.36

Thanks for the tip about disabling webm. At the moment space is not an issue, and yes, since most of my users from Russia, a lot of them do have Yandex Browser. Thus I think I leave both formats for the time being.

**Comment by pedrobmarin** [comment]
Alright. I just pushed a commit that I think fixes this issue 1136db75fc28a3270773e17f58e2134cbd12ec0a.

**Comment by aozalevsky** [comment]
seems to work now. thanks for such a prompt reply!
